api_app/api.py

```diff
@@ -422,7 +423,7 @@ def send_analysis_request(request):
 def get_analyzer_configs(request):
```

Comment:
Move the `get_analyzer_configs` method to `api_app/analyzers_manager/views.py` and register it inside `api_app/analyzers_manager/urls.py`.

Reply:
Infact, I'd prefer to use a class view and not a method view for this so: 

Make a class `AnalyzerListView(generics.ListAPIView)` similar to `ConnectorListView`. ([reerence](https://github.com/intelowlproject/IntelOwl/blob/d9b1872e344d58f76c454106af7fbffe1498d2dc/api_app/connectors_manager/api.py#L10))